glusterd2/transaction/context.go

```diff
@@ -165,6 +167,12 @@ func (c *Tctx) GetNodeResult(peerID uuid.UUID, key string, value interface{}) er
 	return c.Get(storeKey, value)
 }
 
+// GetTxnReqID gets the reqID string saved within the txnCtxConfig.
+func (c *Tctx) GetTxnReqID() string {
+	logFields := c.config.LogFields
```

Comment:
TRIVIAL: Is this temporary variable really needed

Reply:
I will remove the temp variable.